SKILL.tsv

```diff
@@ -4797,175 +4797,175 @@ SKILL_20150717_004796	When an enemy gets shot by [Flash Canon], it will fall int
-SKILL_20150724_004902	Circle 2 Wizard is required
-SKILL_20150724_004903	Circe 3 Wizard is required
-SKILL_20150729_004904	[Capri Sun] Physical/Hit/No Property
-SKILL_20150729_004905	Increase HP, SP restoration
-SKILL_20150729_004906	Restore HP +100{nl}Restore SP +100{nl}Duration -5 sec
-SKILL_20150729_004907	Increase Stats
-SKILL_20150729_004908	Increase All Stats
-SKILL_20150729_004909	Incrase all stats by 10%
-SKILL_20150729_004910	Cooldown time decrease
-SKILL_20150729_004911	Skill cooldown time decreases
-SKILL_20150729_004912	Skill cooldown time decreases by 10%
-SKILL_20150729_004913	Physical ATK, Magic ATK increase
-SKILL_20150729_004914	Physical ATK +10%{nl}Magic ATK +10%
-SKILL_20150729_004915	Physical DEF, Magic DEF increase
-SKILL_20150729_004916	Physical DEF +10%{nl}Magic DEF +10%
-SKILL_20150729_004917	Open shop to repair weapon and temporarily increase its performance.
-SKILL_20150729_004918	Open shop to repair weapon and temporarily increase its performance. Repair is only for Armor, Pants, and Shield.
-SKILL_20150729_004919	Install basecamp for party members. Party members can use basecamp as storage or for teleport. Duration of buffs used around the camp will increase.
+SKILL_20150724_004853	Requires Lv1 Tincturing.
+SKILL_20150724_004854	Requires Lv 3 Kneeling Shot.
+SKILL_20150724_004855	Requires Lv3 Warcry.
+SKILL_20150724_004856	Requires Lv2 Creating zombie.
+SKILL_20150724_004857	Requires Lv3 Creating zombie.
+SKILL_20150724_004858	Requires Lv3 Mackangdel.
+SKILL_20150724_004859	Requires Circle 2 Cataphract.
+SKILL_20150724_004860	Requires Circle 2 Cataphract.
+SKILL_20150724_004861	Requires Circle 2 Chronomancer.
+SKILL_20150724_004862	Requires Lv3 Deprotected zone.
+SKILL_20150724_004863	Requires Lv1 Pyromancer.
+SKILL_20150724_004864	Requires Cryomancer.
+SKILL_20150724_004865	Requires Lv3 Hovering.
+SKILL_20150724_004866	Requires Lv3 Ice wall.
+SKILL_20150724_004867	Requires Lv3 Catar Stroke.
+SKILL_20150724_004868	Requires Cross Guard.
+SKILL_20150724_004869	Requires Circle 2 Highlander.
+SKILL_20150724_004870	Requires Circle 3 Highlander.
+SKILL_20150724_004871	Requires Lv3 Finestra.
+SKILL_20150724_004872	Requires Circle 2 Hunter.
+SKILL_20150724_004873	Requires Lv3 Aukuras.
+SKILL_20150724_004874	Requires Lv3 Handman's Knot.
+SKILL_20150724_004875	Requires Circle 2 Linker.
+SKILL_20150724_004876	Requires Circle 3 Linker.
+SKILL_20150724_004877	Requires Lv3 Restoration.
+SKILL_20150724_004878	Requires Lv2 Turn Undead.
+SKILL_20150724_004879	Requires Lv6 Indulgentia.
+SKILL_20150724_004880	Requires Circle 2 Pardoner.
+SKILL_20150724_004881	Requires Lv3 High Guard.
+SKILL_20150724_004882	Requires Lv6 Umbo Blow.
+SKILL_20150724_004883	Requires Lv6 Rim Blow.
+SKILL_20150724_004884	Requires Circle 2 Priest and Lv2 Resurrection.
+SKILL_20150724_004885	Requires Circle 2 Psychokino.
+SKILL_20150724_004886	Requires Circle 3 Psychokino.
+SKILL_20150724_004887	Requires Lv3 Tear Gas.
```

Comment:
*4887: Not sure if it means to get tear gas at circle 3, or Lv3 tear gas*

Lv. 3 is good.